slack-base/src/main/java/com/hubspot/slack/client/models/events/bot/SlackAppHomeOpenedEventIF.java

```diff
@@ -17,9 +17,9 @@
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 @JsonDeserialize(as = SlackAppHomeOpenedEvent.class)
 public interface SlackAppHomeOpenedEventIF extends SlackEvent, HasUser {
-    String getTab();
+    Optional<String> getTab();
 
-    Optional<HomeTabViewResponse> getView();
+    Optional<HomeTabViewResponseIF> getView();
```

Comment:
We prefer to use the concrete instance of the class, rather than the interface (which just exists so our annotation processor knows to generate the desired POJO)

Reply:
Doh! That was already pointed out by you and corrected by me once before, I just messed up merging an older changeset in. Someday I'll get comfortable with git :) Thanks, I'll fix shortly.